### PR TITLE
chore(snapshots): export block 18010657

### PR DESCRIPTION
**Pull request by albttx**

### CODE REVIEW

**Review by moul — APPROVED** [review]
LGTM, thank you!

**Comment by moul** [comment]
Addresses #18

**Comment by albttx** [comment]
Update, in the block `18010659` we don't have the information we want, updatin to block `18010658`

**Comment by tbruyelle** [comment]
@albttx let's update the PR title to use the correct block height.

**Comment by albttx** [comment]
Ready for review: @jaekwon @moul

**Review by tbruyelle — APPROVED** [review]

**Comment by jaekwon** [comment]
Let's merge it and more eyes will spot any issues. You're all now responsible for the snapshot files and code so please add an entry to the CODEOWNER files as well. I will preclude myself and rely on you for accuracy; what I need is a single spec file that accurately describes what it is and how to use it. 

Then within snapshot/ we can make snapshot/gengenesis.go and snapshot/gengovno.go Go files that can generate genesis files for both chains, or something like that.

We can start with just snapshot/gengovno.go or snapshot/gengovno/* only. The other one, let's leave that for others to figure out later with the help of govno.

**Comment by jaekwon** [comment]
Does it include delegations? Check against AIB address for example: 
What are the steps the duplicate the snapshot from a full node?
Has anyone else verified the steps to verify the snapshot? 
What is the sum of all the ATOMs in the snapshot by vote?

Let's get more eyes on this before merging, just to make sure it doesn't confuse anyone.

**Review by jaekwon — APPROVED** [review]
I added a TODO note that validators that want to join must duplicate and understand these steps. So I validators will do the reviewing for us upon genesis of the chain or govno.

**Comment by giunatale** [comment]
@jaekwon to clarify, the exported json is a standard export of the **state** from a  hub node at that block height.

it's just raw, unprocessed data. But it should contain most everything we need and it should be accurate. (votes and delegations are there)

As written in the readme, due to how tally works and the fact that votes are removed once the tally is done, if we want to be completely accurate we should also look at the content of the next block, which is when the tally is actually performed (but the state exported at that height does not contain votes anymore).

**Comment by Devona0** [comment]
yes

File: snapshots/README.md
Comment (cunum) [inline snapshots/README.md:13]: the file isn't accessible for me, request returns 403:

https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010657.json

Comment (tbruyelle) [reply snapshots/README.md:13]: There was a typo that has been fixed, use the `main` version plz.

**Comment by tbruyelle** [comment]
This document has been moved to https://github.com/atomone-hub/govbox

**Pull request merged**
